graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx

```diff
@@ -18,12 +18,12 @@ class BootstrapModalForm extends Component {
     this.open = this.open.bind(this);
     this.close = this.close.bind(this);
     this._submit = this._submit.bind(this);
-    this._onModalClose = this._onModalClose.bind(this);
+    this._onModalCancel = this._onModalCancel.bind(this);
   }
 
-  _onModalClose() {
-    if (typeof this.props.onModalClose === 'function') {
-      this.props.onModalClose();
+  _onModalCancel() {
+    if (typeof this.props.onCancel === 'function') {
```

Comment:
This change will break some of the other components using `BootstrapModalForm`, as before they relied on the `onModalClose` callback being called when the modal was cancelled or closed, and now it won't be called all the time.

Anyway, why do we need to differentiate between both situations? The way I see it, closing the modal and clicking cancel is the same. Most of the time I also feel that the software I use behaves in that way too (closing a modal is usually triggering the same action as cancelling). That doesn't apply to "reset" buttons, but usually you can't find those in modals.